### PR TITLE
Restart replay

### PR DESCRIPTION
**Pull request by saralein**
Working off exit-game, adds ability to restart or replay the game.

### CODE REVIEW

**Comment by zdavy** [comment]
Can you change this PR to target the `exit-game` branch instead of the `master` branch? It'll show us a smaller isolated set of changes which are specific to the feature.

**Comment by saralein** [comment]
Should be updated now to target exit-game.

File: app.rb [hunk 1]
```diff
@@ -19,14 +20,20 @@ def initializer(size, token1, token2)
   ai = AI.new(checker, token2, token1)
   player1 = Player.new(human, token1, user_interface)
   player2 = Player.new(ai, token2, user_interface)
-  game = Game.new(state, board, checker, player1, player2)
 
   trap_signals(user_interface)
 
-  while(state[:is_playing])
-    state[:is_playing] = false
+  until(state[:stop_playing])
```
Comment (zdavy) [inline app.rb]: can you also access the value via `state.stop_playing`?

Comment (saralein) [reply app.rb]: Yes, thank you very much.  I was wondering about this given one of your other comments.  I was starting right at it in the documentation but missed it.

**Review by zdavy — APPROVED** [review]

**Pull request merged**